target/linux/realtek/files-6.12/drivers/net/phy/rtl83xx-phy.c

```diff
@@ -2855,21 +2855,168 @@ sds_config rtl9300_a_sds_10gr_lane1[] =
 	{0x2B, 0x14, 0x3108}, {0x2D, 0x13, 0x3C87}, {0x2D, 0x14, 0x1808},
 };
 
-static void rtl9300_serdes_patch(int sds_num)
-{
+sds_config rtl9300_a_sds_10g_2500Bx_lane0[] =
+{
+    {0x00, 0x0E, 0x3053},{0x01, 0x14, 0x0100},
+    {0x21, 0x03, 0x8206},{0x21, 0x05, 0x40B0},{0x21, 0x06, 0x0010},{0x21, 0x07, 0xF09F},
+    {0x21, 0x0C, 0x0007},{0x21, 0x0D, 0x6009},{0x21, 0x0E, 0x0000},{0x21, 0x0F, 0x0008},
+    {0x24, 0x00, 0x0668},{0x24, 0x02, 0xD020},{0x24, 0x06, 0xC000},{0x24, 0x0B, 0x1892},
+    {0x24, 0x0F, 0xFFDF},{0x24, 0x12, 0x03C4},{0x24, 0x13, 0x027F},{0x24, 0x14, 0x1311},
+    {0x24, 0x16, 0x00C9},{0x24, 0x17, 0xA100},{0x24, 0x1A, 0x0001},{0x24, 0x1C, 0x0400},
+    {0x25, 0x01, 0x0300},{0x25, 0x02, 0x1017},{0x25, 0x03, 0xFFDF},{0x25, 0x05, 0x7F7C},
+    {0x25, 0x07, 0x8100},{0x25, 0x08, 0x0001},{0x25, 0x09, 0xFFD4},{0x25, 0x0A, 0x7C2F},
+    {0x25, 0x0E, 0x003F},{0x25, 0x0F, 0x0121},{0x25, 0x10, 0x0020},{0x25, 0x11, 0x8840},
+    {0x28, 0x00, 0x0668},{0x28, 0x02, 0xD020},{0x28, 0x06, 0xC000},{0x28, 0x0B, 0x1892},
+    {0x28, 0x0F, 0xFFDF},{0x28, 0x12, 0x01C4},{0x28, 0x13, 0x027F},{0x28, 0x14, 0x1311},
+    {0x28, 0x16, 0x00C9},{0x28, 0x17, 0xA100},{0x28, 0x1A, 0x0001},{0x28, 0x1C, 0x0400},
+    {0x29, 0x01, 0x0300},{0x29, 0x02, 0x1017},{0x29, 0x03, 0xFFDF},{0x29, 0x05, 0x7F7C},
+    {0x29, 0x07, 0x8100},{0x29, 0x08, 0x0001},{0x29, 0x09, 0xFFD4},{0x29, 0x0A, 0x7C2F},
+    {0x29, 0x0E, 0x003F},{0x29, 0x0F, 0x0121},{0x29, 0x10, 0x0020},{0x29, 0x11, 0x8840},
+    {0x2B, 0x13, 0x0050},{0x2B, 0x18, 0x8E88},{0x2B, 0x19, 0x4902},{0x2B, 0x1D, 0x2501},
+    {0x2D, 0x13, 0x0050},{0x2D, 0x18, 0x8E88},{0x2D, 0x17, 0x4109},{0x2D, 0x19, 0x4902},
+    {0x2D, 0x1C, 0x1109},{0x2D, 0x1D, 0x2641},
+    {0x2F, 0x13, 0x0050},{0x2F, 0x18, 0x8E88},{0x2F, 0x19, 0x4902},{0x2F, 0x1D, 0x66E1},
+};
```

Comment:
Reference for reviewers: https://github.com/plappermaul/realtek-doc/blob/54589ff0afa70045abfdc71a3133aa55c76257bc/sources/rtk-xgs1210/src/dal/longan/dal_longan_construct.c#L613